src/main/java/org/apache/commons/io/IOUtils.java

```diff
@@ -2637,57 +2638,61 @@ public static BufferedReader toBufferedReader(final Reader reader, final int siz
     }
 
     /**
-     * Gets the contents of an {@link InputStream} as a {@code byte[]}.
-     * <p>
-     * This method buffers the input internally, so there is no need to use a {@link BufferedInputStream}.
-     * </p>
+     * Reads all remaining bytes from the given {@link InputStream} into a new {@code byte[]}.
      *
-     * @param inputStream the {@link InputStream} to read.
-     * @return the requested byte array.
-     * @throws NullPointerException if the InputStream is {@code null}.
-     * @throws IOException          if an I/O error occurs or reading more than {@link Integer#MAX_VALUE} occurs.
+     * <p>The method accumulates the data in temporary buffers and returns a single array
```

Comment:
This comment is not needed IMO, the user cannot affect the behavior about it by passing in different values. It's a low-level implementation detail.

Reply:
Removed in https://github.com/apache/commons-io/pull/776/commits/97d37a94de8c935ea8e4961e354a1cd1d5c49b6a